### PR TITLE
[aot] Add script to generate Taichi C++ wrapper for Unreal Engine

### PR DESCRIPTION
**Pull request by jim19930609**
Issue: #

### CODE REVIEW

File: misc/generate_taichi_unreal_header.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Generated by GPT-4 with minor modifications
```
Comment (PENGUINLIONG) [inline misc/generate_taichi_unreal_header.py:1]: lmao wtf

File: misc/generate_taichi_unreal_header.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    with open(file_path, 'r') as f:
+        lines = f.readlines()
+
+    include_section_end = 0
+    for index, line in enumerate(lines):
+        if line.startswith('#include'):
+            include_section_end = index
+
+    macro_def = '''
+#include "AOTPlugin.h"
```
Comment (PENGUINLIONG) [inline misc/generate_taichi_unreal_header.py:18]: Is this general enough?

**Review by PENGUINLIONG — APPROVED** [review]
Not sure how it works but please feel free to merge if it suites your need. :)

File: misc/generate_taichi_unreal_header.py (hunk 2 above)
Comment (jim19930609) [reply misc/generate_taichi_unreal_header.py:18]: Nope, it's actually designed to be used in combination with: https://github.com/jim19930609/taichi-unreal. Or maybe I should put this script in the taichi-unreal repo?

**Comment by jim19930609** [comment]
Let me put this script to `https://github.com/jim19930609/taichi-unreal` instead, since it has to be used in sync with what's defined in the taichi-unreal repo

**Pull request closed without merging**